<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qschedule</class>
 <widget class="QWidget" name="Qschedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Qschedule{
	border-image: url(:/Images/55/6462-11102Q54A389.jpg);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>140</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/cjh/5445514_151421049066_2.jpg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextBrowser" name="timeTextBrowser">
           <property name="minimumSize">
            <size>
             <width>375</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>375</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/cjh/112.jpg);</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>375</width>
               <height>105</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>375</width>
               <height>105</height>
              </size>
             </property>
             <property name="title">
              <string> 今日安排记事</string>
             </property>
             <widget class="QTextBrowser" name="eventTextBrowser">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>22</y>
                <width>355</width>
                <height>85</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>355</width>
                <height>85</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>355</width>
                <height>85</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <zorder>eventTextBrowser</zorder>
             <zorder>calendarWidget</zorder>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="allBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/cjh/粉红1.png);</string>
         </property>
         <property name="text">
          <string>全部备忘</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pDlt">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/cjh/粉红1.png);</string>
         </property>
         <property name="text">
          <string>删除备忘</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/cjh/粉红1.png);</string>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
